pkg/storaged/stratis-details.jsx

```diff
@@ -641,20 +642,32 @@ export const StratisPoolDetails = ({ client, pool }) => {
         return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
     }
 
-    let alert = null;
+    const alerts = [];
     if (client.features.stratis_grow_blockdevs &&
         blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize))) {
-        alert = (<Alert key="unused-space"
-                        isInline
-                        variant="warning"
-                        title={_("This pool does not use all the space on its block devices.")}>
+        alerts.push(<Alert key="unused-space"
+                           isInline
+                           variant="warning"
+                           title={_("This pool does not use all the space on its block devices.")}>
             {_("Some block devices of this pool have grown in size after the pool was created. The pool can be safely grown to use the newly available space.")}
             <div className="storage_alert_action_buttons">
                 <StorageButton onClick={grow_blockdevs}>{_("Grow the pool to take all space")}</StorageButton>
             </div>
         </Alert>);
     }
 
+    if (pool.AvailableActions !== "fully_functional") {
+        const goToStratisLogs = () => cockpit.jump("/system/logs/#/?prio=warn&_SYSTEMD_UNIT=stratisd.service");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18843-20230829-061921-68e1f0e2-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)